repair/row_level.cc

```diff
@@ -2486,6 +2486,11 @@ class row_level_repair {
                 }
                 rlogger.debug("Called master.get_sync_boundary for node {} sb={}, combined_csum={}, row_size={}, zero_rows={}, skipped_sync_boundary={}",
                     node, res.boundary, res.row_buf_combined_csum, res.row_buf_size, _zero_rows, _skipped_sync_boundary);
+            }).handle_exception([this, node] (std::exception_ptr ep) {
+                auto s = _cf.schema();
+                rlogger.warn("repair[{}]: get_sync_boundary: got error from node={}, keyspace={}, table={}, range={}, error={}",
```

Comment:
Why `warn`, why not `error`?

Reply:
This could be a minor issue, e.g., a temporary network failure, which is not something we can not recover. Using warn is enough IMO.